src/overlays/actors/ovl_Arrow_Ice/z_arrow_ice.h

```diff
@@ -9,7 +9,12 @@ typedef void (*ArrowIceActionFunc)(struct ArrowIce*, GlobalContext*);
 
 typedef struct ArrowIce {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x1C];
+    /* 0x0144 */ s16 radius;
+    /* 0x0146 */ u16 timer;
+    /* 0x0148 */ u8 alpha;
+    /* 0x014C */ Vec3f unkPos;
+    /* 0x0158 */ f32 unk_158;
```

Comment:
This looks to be the scale used when the arrow effect is growing.

`height` is probably the right term since this is a cone.